packages/ui/react-ui-kanban/src/defs/kanban.ts

```diff
@@ -34,8 +33,8 @@ export const KanbanSchema = S.Struct({
 });
 
 export const KanbanPropsSchema = S.Struct({
```

Comment:
it there a better name than props? why isn't this part of KanbanSchema?

Reply:
Renamed to `KanbanSettingsSchema`